src/registry_schemas/example_data/schema_data.py

```diff
@@ -2343,19 +2343,37 @@
 
 CONTINUATION_IN = {
     'business': {
-        'identifier': 'BC1234567',  # Identifier of the registered extra provincial business
+        'identifier': 'A1234567',  # Identifier of the registered extra provincial business
         'taxId': '123456789'  # Existing BN Number if any
```

Comment:
Since line 2352 is the legal name in the foreign jurisdiction, should we add `business.legalName` here for the legal name in BC?

Also, why is taxId here and also on line 2355?

Reply:
that make sense. will update the sample data. and we don't need taxId in line 2347